DokkaRoot.md

```diff
@@ -0,0 +1,181 @@
+# Official Android SDK for [Stream Chat](https://getstream.io/chat/sdk/android/)
+
+> This is the official Android SDK for [Stream Chat](https://getstream.io/chat/sdk/android/), a service for building chat and messaging applications. This library includes both a low-level chat SDK and a set of reusable UI components. Most users start with the UI components, and fall back to the lower level API when they want to customize things.
+
+<br />
+<p align="left">
+  <a href="https://github.com/GetStream/stream-chat-android/actions"><img src="https://github.com/GetStream/stream-chat-android/workflows/Build%20and%20test/badge.svg" /></a>
+  <a href="https://android-arsenal.com/api?level=21"><img alt="API" src="https://img.shields.io/badge/API-21%2B-brightgreen.svg?style=flat"/></a>
+  <a href="https://github.com/GetStream/stream-chat-android/releases"><img src="https://img.shields.io/github/v/release/GetStream/stream-chat-android" /></a>
+</p>
+
+### 🔗 Quick Links
+
+* [Register](https://getstream.io/chat/trial/): Create an account and get an API key for Stream Chat
+* [Chat Tutorial](https://getstream.io/tutorials/android-chat/#kotlin): Learn the basics of the SDK by by building a simple messaging app (Kotlin or Java)
+* [UI Components sample app](/stream-chat-android-ui-components-sample): Full messaging app with threads, reactions, optimistic UI updates and offline storage
+* [Compose UI Components sample app](/stream-chat-android-compose-sample): Messaging sample app built with Jetpack Compose!
+* [Client Documentation](https://getstream.io/chat/docs/android/?language=kotlin)
+* [UI Components Documentation](https://getstream.io/chat/docs/sdk/android/)
+* [Compose UI Components Documentation](https://getstream.io/chat/docs/sdk/android/compose/overview/)
+* [API docs](https://getstream.github.io/stream-chat-android/): Full generated docs from Dokka
+* [Jetpack Compose Planning](https://github.com/orgs/GetStream/projects/6): Jetpack Compose public project management board and milestone overview
```

Comment:
This shouldn't be Jetpack Compose, but rather UI Team, also, we can feature Core planning too!